tut/stream-client.md

```diff
@@ -0,0 +1,174 @@
+This tutorial guides you through the process of setting up a web-based streaming client using the Anbox Cloud streaming stack.
+
+## Preparation
+
+### Install the Anbox Cloud Appliance
+We need the Anbox Cloud streaming stack to be deployed already to set up a streaming client. So let's get the streaming stack ready by installing the Anbox Cloud Appliance. Follow the instructions in the [appliance installation tutorial](https://discourse.ubuntu.com/t/22681) until you finish initialising the Appliance.
+
+### Create an access token
+
+To access the HTTP API of the Anbox Stream Gateway, an access token is required. Each access token is associated with a service account.
+
+On the machine where Anbox Cloud Appliance is installed, create the service account by running the following command:
+
+    anbox-cloud-appliance gateway account create test
+
+The output of this command provides the access token. Make a note of this token to use when you make a request to the stream gateway API.
+
+### Create an application
+
+Follow the instructions in [How to create an application](https://discourse.ubuntu.com/t/24198) and create an application.
+
+## Set up the stream client
+Create a directory to set up the stream client:
+
+    sudo mkdir -p /srv/stream-client
+
+Create a `demo.html` file inside `/srv/stream-client` with the following content:
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+    <meta content="utf-8" http-equiv="encoding">
+    <title>Anbox Streaming SDK Example</title>
+</head>
+<body>
+    <script type="module">
+    import {AnboxStream, AnboxStreamGatewayConnector} from './anbox-stream-sdk.js';
+
+    const connector = new AnboxStreamGatewayConnector({
+        // Run `juju status` to get the stream gateway IP/domain name
+        // Replace 'https://gateway.url.net' with the stream gateway IP/domain name
+        url: 'https://gateway.url.net',
+        //Use the access token created earlier as 'YOUR_AUTH_TOKEN'
+        authToken: 'YOUR_AUTH_TOKEN',
+        session: {
+            // Run `amc list` to see a list of all applications
+            // Replace `com.foo.bar` with your application name
+            app: "com.foo.bar",
+        },
+        // Set the display resolution and frame rate for the Android container
+        screen: {
+            width: 1280,
+            height: 720,
+            fps: 25,
```

Comment:
Should we also indicate that these can be adjusted?

Reply:
I have done so now in comments, we can take the same approach as for the earlier URL, token and app name in case we go with text explanation